web/moduly/program-nahled-api.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$this->bezStranky(true);
+
+if(($idAktivity = get('idAktivity'))) {
+  $a = Aktivita::zId($idAktivity);
+
+  header('Content-type: application/json');
+  $config = JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES;
```

Comment:
Nepřepíšeme tímhle nějakou globální proměnnou? Ten název `$config` je hodně generický, je velká šance na kolizi. Nedáme tohle nastavení rovnou do funkce? Zas tak dlouhé to nebude: `json_encode([], JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES)`

Reply:
Ty moduly jsou wrapnute v metode aby se to nedelo, takze by to nemelo hrozit. Obecne by to nemelo s globalnim scope nijak interagovat.

Reply:
Vida, to je dobré.